### PR TITLE
lgtm: Install a newer Meson version using pip

### PR DESCRIPTION
**Pull request by smcv**
Loosely based on a version formerly used by systemd.

Might fix https://github.com/hughsie/libjcat/issues/21 or not, but I can't really test it without opening a PR :-)

### CODE REVIEW

File: lgtm.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+extraction:
```
Comment (smcv) [inline lgtm.yml]: This could be a hidden file `.lgtm.yml` if you'd prefer. Those seem to be the only two filenames allowed.

Comment (hughsie) [reply lgtm.yml]: The hidden one please. I did try something like this a few weeks ago but my Debian-foo wasn't strong enough to get over the pip warning you'll probably get. Thanks for looking at this!

**Comment by smcv** [comment]
This might work, but I'm flying blind, because lgtm doesn't seem to want to show me build logs for successful builds... and it won't be happy with a PR unless both the base commit and the PR build successfully, so when I get this working, you'll have to ignore the failure and merge it anyway.

**Comment by smcv** [comment]
https://lgtm.com/projects/g/hughsie/libjcat/logs/rev/pr-23e0c6e173a0e6c5c0e77a2a197e82ff31ea5880/lang:cpp/stage:Build%20master_d6dc3e90b0c805cc50bff9d1d1b87ff575e53769 says this was successful, I think? (But I've never used lgtm before so I don't really know.)

You'll probably want to do something similar for any projects that vendor libjcat into their CI.

**Comment by hughsie** [comment]
I still see `[2020-03-30 23:56:46] [build] meson.build:1:0: ERROR: Meson version is 0.51.2 but project requires >=0.52.0` -- doesn't meson have to be installed system-wide?

**Comment by smcv** [comment]
I think that's for the step where we built master at commit d6dc3e9, in the state it had *before* merging this branch. That's what I meant when I said "it won't be happy with a PR unless both the base commit and the PR build successfully".

I think what I'm doing (and, again, I can't actually tell, because lgtm won't show me a log in the case where I succeed!) is to install meson to `~/.local/bin/meson`, and then add that to PATH, so that when lgtm's build-system abstraction runs `meson`, it finds the version in `~/.local/bin/meson` and runs that to build libjcat.

**Comment by smcv** [comment]
In my original version of this PR, when I just tried to install `meson` rather than `meson >= 0.52.0`, I was seeing both `Build merge (xxxxxxx)` and `Build master (xxxxxxx)` fail. Now, I see `Build merge` succeed, and only `Build master` fails. So I *think* I have solved the problem - but we won't know for sure until you merge this, and someone sends another PR that lgtm will try building.

Sorry, I think that's the best I can do.

**Comment by hughsie** [comment]
Lets see what explodes! Thanks.

**Comment by smcv** [comment]
> until you merge this, and someone sends another PR that lgtm will try building

#26 is a suitable other PR.

**Pull request merged**